ddl/db_test.go

```diff
@@ -450,7 +450,7 @@ func (s *testDBSuite) TestCancelDropIndex(c *C) {
 			rs.Close()
 		}
 		t := s.testGetTable(c, "t")
-		indexInfo := schemautil.FindIndexByName("idx_c2", t.Meta().Indices)
+		indexInfo := expression.FindIndexByName("idx_c2", t.Meta().Indices)
```

Comment:
I was wondering why change `schemautil` to `expression`?

Reply:
There is only one function `FindIndexByName`  in  `schemautil`, after talk with @zz-jason , we decided to move this function to expression.

Reply:
@crazycs520 @xiekeyi98 PTAL this comment https://github.com/pingcap/tidb/pull/9850#discussion_r269843007

Reply:
I will do this in the next PR.